### PR TITLE
DE updated

### PR DESCRIPTION
**Pull request by zRxnx**
German language updated

### CODE REVIEW

File: language/de.json [hunk 1]
```diff
@@ -161,15 +161,15 @@
...
-    "peds": "Peds",
-    "props": "Props",
-    "cleaningcar": "Lösche Auto %s",
-    "cleaningped": "Lösche Ped %s",
-    "cleaningprop": "Lösche Objekt %s",
+    "cleanareaguide": "Entfernt alle Entities in der nähe.",
+    "cars": "Fahrzeuge",
+    "peds": "NPC's",
+    "props": "Objekte",
+    "cleaningcar": "Lösche Fahrzeuge %s",
```
Comment (Blumlaut) [inline language/de.json:168]: these 3 are wrong, it should be individual "Fahrzeug", "NPC" and "Objekt", because the "%s" is their respective id.

Comment (zRxnx) [reply language/de.json:168]: hö?
I only changed the Name

**Review by Blumlaut — CHANGES_REQUESTED** [review]

**Comment by zRxnx** [comment]
image

Fixed to singular

**Comment by zRxnx** [comment]
oh wait sorry xD

**Comment by zRxnx** [comment]
Should be fixed

**Pull request merged**